en/Editing and formatting/Obsidian Flavored Markdown.md

```diff
@@ -0,0 +1,7 @@
+Obsidian strives for maximum capability without breaking any existing formats. As a result, we use a slightly unorthodox combination of flavors of Markdown.
+
+Obsidian mainly uses [CommonMark](https://commonmark.org/), with some functionality from [GitHub Flavored Markdown](https://github.github.com/gfm/) and LaTeX.
+
+In addition, Obsidian adds syntax for [[Internal links]] (`[[ ]]`) and for [[Embedding files]] (`![[ ]]`) syntax.
+
+To be able to handle huge files, and to support syncing between editor views, Obsidian doesn't support Markdown syntax or blank lines within HTML blocks.
```

Comment:
Could you verify this? @lishid I'm not sure if that's the motivation behind whether to handle HTML blocks.